common/config/rush/command-line.json

```diff
@@ -70,7 +70,7 @@
       "name": "build:all-flavors",
       "summary": "Build all flavor in one command (mainly for api update)",
       "description": "Build all flavor in one command (mainly for api update). This command will switch current flavor to beta.",
-      "shellCommand": "(rush switch-flavor:stable && rush rebuild) || rush switch-flavor:beta && rush rebuild",
+      "shellCommand": "(rush switch-flavor:stable && rush rebuild -v) || rush switch-flavor:beta && rush rebuild -v",
```

Comment:
This won't fix when we do `rush build -t <pkg_name>` will it?

Reply:
The problem is when rush runs multiple builds in parallel (so `rush build -t` doesn't log verbose either). But `rushx build` will give you verbose errors (or run with `rush build -t -v`

Reply:
but `rush build -t` isn't running multiple builds in parallel is it?

Reply:
(or at least no more than it was before conditional compile)

Reply:
No, since it is a pure rush command and we can't override it. Only solution is to add a -v flag :)

Reply:
My concern with `-v` is the output can get out of hand quickly - it's use isn't for seeing errors that should be reported as part of the normal build process it's, typically, for delving deeper into why there was a failure.
I don't think we understand why our errors are being silenced by the new build commands well enough to say this is the only solution (it might be I just haven't been convinced on it)

Reply:
Sure, that's a totally valid point @JamesBurnside 
This just seemed like the quickest easiest way to fix it.
We can always look for answers later if this unblocks us.

Also to your point about `-v` getting out of hand.
We use this flag extensively in our github pipelines.
I am yet to see the output get out of control :D

Reply:
We talked before about our build system being scary, adding _even more_ output to that only furthers that.
For a simply mistyped variable name the error is super hidden in amongst other errors and red text:
![image](https://user-images.githubusercontent.com/2684369/151616088-d47552d7-2667-49fb-aebd-af2ee4d21624.png)
(note this is only the latter half the output also)

Reply:
`grep "error"` FTW :D

Reply:
About the use of `-v` in other commands in the pipeline -- I recently added it, and not with a lot of thought, so that precedent isn't worth much.

Open to better options. I just landed that PR to get errors from CI where there were none.